zen-news-summarization/README.md

```diff
@@ -0,0 +1,327 @@
+information on how you can achieve do that, please check 
+[the corresponding docs page](https://docs.zenml.io/getting-started/deploying-zenml).
+
+## ZenNews Stack
+
+Once the ZenML is deployed, we can start to build up our stack. Our stack will 
+consist of the following components:
+
+- [GCP Secrets Manager](https://docs.zenml.io/component-gallery/secrets-managers/gcp)
+- [GCP Container Registry](https://docs.zenml.io/component-gallery/container-registries/gcloud)
+- [GCS Artifact Store](https://docs.zenml.io/component-gallery/artifact-stores/gcloud-gcs)
+- [Vertex Orchestrator](https://docs.zenml.io/component-gallery/orchestrators/gcloud-vertexai)
+- [Discord Alerter (part of the `zennews` package)](src/zennews/alerter/discord_alerter.py)
+ 
+Let's start by installing the `gcp` integration:
+
+```bash
+zenml integration install gcp
+```
+
+### Secrets Manager
+
+The first component to register is a
+[GCP secrets manager](https://docs.zenml.io/component-gallery/secrets-managers/gcp). 
+The command is quite straightforward. You just have to give it a name and 
+provide the ID of your project on GCP.
+
+```bash
+zenml secrets-manager register <SECRETS_MANAGER_NAME> \
+    --flavor=gcp \
+    --project_id=<PROJECT_ID>
+```
+
+### Container Registry
+
+The second component is a 
+[GCP container registry](https://docs.zenml.io/component-gallery/container-registries/gcloud). 
+Similar to the previous component, you just need to provide a name and the 
+URI to your container registry on GCP.
+
+```bash
+zenml container-registry register <CONTAINER_REGISTERY_NAME> \
+    --flavor=gcp \
+    --uri=<REGISTRY_URI>
+```
+
+### Artifact Store
+
+The next component on the list is a 
+[GCS artifact store](https://docs.zenml.io/component-gallery/artifact-stores/gcloud-gcs). 
+In order to register it, all you have to do is to provide the path to your GCS
+bucket:
+
+```bash 
+zenml artifact-store register <ARTIFACT_STORE_NAME> \
+    --flavor=gcp \
+    --path=<PATH_TO_BUCKET> 
+```
+
+### Orchestrator
+
+Following the artifact store, we will register a
+[Vertex AI orchestrator.](https://docs.zenml.io/component-gallery/orchestrators/gcloud-vertexai)
+
+```bash
+zenml orchestrator register <ORCHESTRATOR_NAME> \
+    --flavor=vertex \
+    --project=<PROJECT_ID> \
+    --location=<GCP_LOCATION> \
+    --workload_service_account=<EMAIL_OF_YOUR_SERVICE_ACCOUNT> \
+    --service_account_path=<PATH_TO_YOUR_SERVICE_ACCOUNT_KEY>
+```
+
+You need to simply provide the id of your project, the name of your GCP 
+region and the service account you would like to use.
+
+> Warning: In this version, you have to provide both the email of the service 
+> account and the path to a key.json file. This interaction will be improved 
+> with the upcoming releases.
+
+Make sure that the service account has the proper roles for the following 
+services: Cloud Functions, Cloud Scheduler, Secret Manager, Service Account,
+Storage, and Vertex AI,
```

Comment:
```suggestion
Storage, and Vertex AI.
```

This feels a bit unclear. "the proper roles" might need to be specified and defined here?

Reply:
I left it a little bit vague on purpose, as this might depend on the person who is running the example. For instance, when I was running it, I mostly used `admin` roles for the GCP services but others might want to avoid that.